themes/mlibrary_new/exhibit-builder/exhibits/summary.php

```diff
@@ -13,19 +13,17 @@
 
 ?>
 <!--Breadcrumb and Share Bar-->
-  <div class="col-xs-9">
+  <div class="col-xs-12 col-sm-8">
       <ol class="breadcrumb">
         <li class="breadcrumb-item"><?php echo link_to_home_page(__('Home')); ?></li>
         <li class="breadcrumb-item active"><?php echo metadata('exhibit','title',array('no_escape' => true)); ?></li>
       </ol>
   </div>
 
-  <div class="col-xs-3">
+  <div class="col-xs-12 col-sm-4">
     <ul class="share-this">
       <li>Share this Exhibit:</li>
-      <li><div class="fb-like" data-send="false" data-layout="button_count" data-show-faces="false" data-font="arial"></div></li>
-      <li><div class="twitter-share">
-      <a  href="https://twitter.com/share" class="twitter-share-button" data-text="I just saw '<?php echo metadata('exhibit','title',array('no_escape' => true)); ?>' at the U-M Library Online Exhibits!" > Tweet</a></div></li>
+      <li><a href="https://twitter.com/share" class="twitter-share-button" data-text="I just saw '<?php echo metadata('exhibit','title',array('no_escape' => true)); ?>' at the U-M Library Online Exhibits!" ><span class="sr-only">Tweet</span> </a></li>
```

Comment:
I can see that it was there before, but the `'no_escape' => true` bit is concerning to me.  Has this been tested with titles that feature quotation marks?

Reply:
We can definitely open a ticket for that. I brought that over from the old theme, so I'm not sure if that was addressed before.